src/app/authentication/services/auth-service.service.ts

```diff
@@ -0,0 +1,31 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { IUserEntyty } from 'src/app/interfaces/user-entyty.interface';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthServiceService {
+
+  usersData: IUserEntyty[]
+
+  constructor(private http: HttpClient) { }
+
+  async logIn(): Promise<any> {
+    let userData = await fetch('http://localhost:3000/users');
```

Comment:
you should know what specific endpoint returns
e.g. IUserEntyty[] or something else
also you don't need async/await here
```
logIn(): Promise<IUserEntyty[]> {
  return fetch('http://localhost:3000/users').then((data) => data.json());
}
```